src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -152,23 +152,23 @@ public function setAllowOverwrite($allow)
     }
 
     /**
-     * Sets the closures used for normalization.
+     * Sets the callbacks used for normalization.
      *
-     * @param \Closure[] $closures An array of Closures used for normalization
+     * @param callable[] $callbacks An array of callbacks used for normalization
      */
-    public function setNormalizationClosures(array $closures)
+    public function setNormalizationCallbacks(array $callbacks)
```

Comment:
but in this case here it is probably better to deprecate the method in 2.8 first and provide the new method. so there is an upgrade path.